packages/webcert/src/feature/certificate/Inputs/UeCauseOfDeath/UeCauseOfDeathControl.tsx

```diff
@@ -77,6 +81,9 @@ const UeCauseOfDeathControl: React.FC<Props> = ({
     ? (validation.find((v) => v.type === CertificateDataValidationType.TEXT_VALIDATION) as TextValidation)
     : undefined
 
+  const emptyValidationError = validationErrors ? (validationErrors.find((e) => e.type === 'EMPTY') as ValidationError) : undefined
+  const nonEmptyValidationErrors = validationErrors ? validationErrors.filter((e) => e.type !== 'EMPTY') : undefined
+
   const specifications: ConfigUeDropdownItem[] = [{ id: '', label: 'Välj...' }, ...config.specifications]
```

Comment:
This needs to be changed to `specifications: ConfigUeCodeItem[]` with a code value of `''` for the first option.